backend/de.metas.business/src/main/sql/postgresql/ddl/views/c_bpartner_adv_search_v.sql

```diff
@@ -1,38 +1,37 @@
 DROP VIEW IF EXISTS C_BPartner_Adv_Search_v
 ;
 
-CREATE OR REPLACE VIEW C_BPartner_Adv_Search_v AS
+CREATE VIEW c_bpartner_adv_search_v(c_bpartner_id, c_bpartner_location_id, c_bp_contact_id, value, externalid, iscompany, name, firstname, lastname, companyname, address1, city, postal, ad_client_id, ad_org_id, isactive, created, createdby, updated, updatedby, es_documentid) AS
```

Comment:
pls compare with the left side and perform only the really needed changes?
it's quite hard for me to understand what exactly changed